roles/falcon_installation/tasks/api.yml

```diff
@@ -22,9 +22,41 @@
       Content-Type: application/json
   register: falcon_api_target_cid
 
+# Block when falcon_sensor_update_policy_name is supplied
+- block:
+  - name: "CrowdStrike Falcon | Build Sensor Update Policy API Query"
+    set_fact:
+      falcon_sensor_update_policy_query: "{{ 'platform_name:\"' + falcon_sensor_update_policy_platform + '\"+name.raw:\"' + falcon_sensor_update_policy_name + '\"' }}"
+
+  - name: "CrowdStrike Falcon | Search for Sensor Update Policy"
+    uri:
+      url: "https://{{ falcon_cloud }}/policy/combined/sensor-update/v2?filter={{ falcon_sensor_update_policy_query | urlencode }}"
+      method: GET
+      return_content: true
+      headers:
+        authorization: "Bearer {{ falcon_api_oauth2_token.json.access_token }}"
+        Content-Type: application/json
+    register: falcon_sensor_update_policy_info
+
+  - name: "CrowdStrike Falcon | Validate Sensor Update Policy request"
+    fail:
+      msg: "No Falcon Sensor Update Policy with name: {{ falcon_sensor_update_policy_name }} was found!"
+    when: falcon_sensor_update_policy_info.json.resources[0] is not defined
+
+  - name: "CrowdStrike Falcon | Get the Falcon Sensor version from Update Policy"
+    set_fact:
+      falcon_sensor_update_policy_package_version: "{{ falcon_sensor_update_policy_info.json.resources[0].settings.sensor_version }}"
```

Comment:
This result is incomplete as it doesn't handle `aarch64` systems.

Reply:
We are going to add this functionality in a follow on PR that enables arm functionality.